### ISSUE TITLE
Adjusting game speed when in local multiplayer match

### ISSUE THREAD

**Issue by ptkk73** [Sh0wdown/UnofficialCrusaderPatch#548]
Is that even possible to do? I mean increasing the speed during the actual game. 

Thanks.

**Comment by PodeCaradox** [comment]
I think that's not possible, because the games need to be in sync, and the speed increase is maybe not send over the network.

**Comment by ptkk73** [comment]
Yes, that's what I think too. Just wanted to make sure if there is any way to make the match more difficult once it gets just too easy and winning is just a matter of time

**Comment by J-T-de** [comment]
As PodeCaradox commented, it is not feasable to do this with some kind of player input (who even should have the power to decide a speed change).

What should be possible is to trigger speed changes at fixed ingame times (e.g. a slower setup phase and after X monthes a faster fighting phase) or other fixed events (like the first player has 100 units or 5000 gold or so). Possible and kinda feasable.

**Comment by ptkk73** [comment]
Well, that is a great idea. Definitely would love to see something like that :) Thanks for the responses

**Comment by realPhint** [comment]
Maybe it could become possible, when implementing an option to pause multiplayer games like for example in warcraft 3 (every player gets 3 timeouts for him to use). if thats possible there could be an option to "vote" for a new game speed.
for example: you have a button ingame opening a window where you can put in the game speed you are suggesting and upon suggesting the game pauses and every player gets to vote for accepting or not accepting the new speed. this of course should have a 10min cooldown or so, to make it unabusable

**Comment by J-T-de** [comment]
It's basically the same problem: It is not clear how to communicate, and adding UI-stuff is quite hard. So also rather unfeasible...
